app/resources/admin.html.template

```diff
@@ -231,67 +231,183 @@
+          <div>
+            <strong>Effect:</strong> The repository is listed on
+            <a href="{{env.global_url}}" target="_blank">the home page</a> and
+            <a href="{{env.global_url}}/feeds/repo" target="_blank">the
+            repository feed</a>. So end users can discover the repository.
+          </div>
+        </div>
+      </div>
+      <div class="option">
+        <input
+            type="radio" name="launch_status" value="deactivated"
+            id="launch_status_deactivated"
+            {% if view_config.launch_status == "deactivated" %}
+              checked
+            {% endif %}>
+        <label for="launch_status_deactivated">
+          Deactivated
+        </label>
+        <div class="note">
+          <div>
+            <strong>Use case:</strong> Switch to this mode to deactivate the
+            repository.
+          </div>
+          <div>
+            <strong>Effect:</strong> Replaces content of all pages with the
+            deactivation message below. People cannot interact with the
+            repository either via web interface or API. The repository is
+            neither listed on
+            <a href="{{env.global_url}}" target="_blank">the home page</a> nor
+            <a href="{{env.global_url}}/feeds/repo" target="_blank">the
+            repository feed</a>.
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="config">
+      <label for="deactivation_message_html">
+        Deactivation message (HTML):</label>
+      <div class="response">
+        <textarea name="deactivation_message_html"
+            id="deactivation_message_html" rows=6 cols=80
+            >{{view_config.deactivation_message_html|default:""}}</textarea>
+      </div>
+    </div>
+
     <div class="config">
       <div class="option">
         <input type="radio" name="test_mode" value="true"
           id="test_mode_true"
-          {{config.test_mode|yesno:"checked,"}}>
+          {{view_config.test_mode|yesno:"checked,"}}>
         <label for="test_mode_true">
           Test mode
         </label>
-	<div class="note">
-	  In this mode, all records are deleted about
-	  {{test_mode_min_age_hours}} hours after they are created, regardless
-	  of their specified expiry date.  If you switch from real mode to
-	  test mode, existing records that are more than
-	  {{test_mode_min_age_hours}} old will soon be deleted.
-          Note that (unlike regular deletions or expirations)
-          these deletions do not generate expiry notices
-          in the outgoing PFIF feeds,
-          so any clients that import test records into their own repositories
-          cannot be expected to delete them on the same short schedule.
-	</div>
+        <div class="note">
+          <div>
+            <strong>Use case:</strong> This mode is usually used for always-on
+            repository such as "japan" repository, while no crisis is
+            on-going, so it is only used for testing purpose.
```

Comment:
maybe drop ", so it is only used for testing purpose". had a slight difficulty in parsing.